src/main/java/com/upsolver/datasources/jdbc/JDBCDataSource.java

```diff
@@ -121,9 +121,7 @@ public void setProperties(Map<String, String> properties) {
         var rs = metaData.getTableTypes();
         while (rs.next()) {
             var type = rs.getString("TABLE_TYPE").toUpperCase();
-            if (type.equals("TABLE") /*|| type.equals("SYNONYM")*/) {
-                result.add(type);
-            }
+            result.add(type);
```

Comment:
This won't work out of the box. For example I tried with Synonym in Oracle and it didn't work